src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,112 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+import numpy as np
+
+def read_eleanorlite_lightcurve(filename,
+    flux_column="CORR_FLUX",
+    quality_bitmask="default", 
+    **kwargs):
```

Comment:
What is the use case for the optional `**kwargs`? The other readers (TESS, QLP, etc.) don't have it.

If there is some use case, it needs to be documented.

If there is no use case, let's take it out of the function signature.